src/app/docs/action-button/action-button-docs.component.html

```diff
@@ -4,10 +4,130 @@
+              </li>
+            </ul>
+          <li>
+            Methods of completion are system-defined, not user-defined.
+          </li>
+          <li>
+            Methods of completion mostly determine how tasks proceed.
+          </li>
+            <ul>
+              <li>
+                This choice is made at or near the beginning of the task
+              </li>
+            </ul>
+          <li>
+            The outcome of end result will be largely the same, regardless of the method
+          </li>
+        </ul>
+      <p class="sky-docs-text-block">
+        Action buttons represent a decision point within a task where users must select a method of completion. Do not use them when users can skip them, avoid them, or address them later. Present action buttons in a way that isolates the choices that they represent and removes other decisions, tasks, or content. Use the details line on the buttons to provide context or explain differences between the options.
+      </p>
+      <p class="sky-docs-text-block">
+        Use action buttons immediately after users initiate tasks to differentiate the methods of completion.
+      </p>
+      <p class="sky-docs-text-block">
+        Use action buttons on pages or modals when tasks are underway and the only relevant choice for users is how to complete tasks. Do not display other inputs or choices until users select an action button. Then hide the action buttons or close the modals.
+      </p>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Don't use when"
+      headingStyle="danger"
+    >
+      <p class="sky-docs-text-block">
+        Don't use an action button by itself. Action buttons should not serve as primary calls-to-action on pages where other actions are available to users.
+      </p>
+      <p class="sky-docs-text-block">
+        Do not use actions buttons to present different methods of completing tasks before users initiate the tasks. Action buttons should not compete against other content for user attention.
+      </p>
+      <p class="sky-docs-text-block">
+        Do not use action buttons when the methods of completion in tasks differ only in minor ways such as an extra field or two. Action buttons represent consequential decisions where the methods of completion can start and end differently and can include entirely different steps and requirements. Likewise, do not use action buttons when consequential choices are not made at the beginning of tasks. Instead, use a different selection input and progressive disclosure to reveal additional content as necessary.
+      </p>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior and states"
+    >
+      <p class="sky-docs-text-block">
```

Comment:
Do you think we should add a secondary `Task placement` header to mimic the current docs? Might not be necessary, but we generally have subheaders in this section. We'd just need to wrap the content in `sky-docs-design-guideline` elements and add `heading="Task placement"`.

Regardless, I think we need to add `class="sky-docs-text-block"` to the items here where it is missing. Some of the content in this section is much wider than the limit that class applies.

Reply:
Added the subheader but didn't fix the list items that are extra long because the content in that section isn't typically formatted in bulleted lists. It didn't seem worth adding a p tag on each list item that will likely change to be shorter in the future.